Add TensorFlow image classification example

VanDavv opened the pull request:
# Gen2 Tensorflow Image Classification example

This example demonstrates how to run a neural network created using [TensorFlow Image Classification tutorial](https://www.tensorflow.org/tutorials/images/classification)
(which one of our community members has put together in [a single Colab Notebook](https://colab.research.google.com/drive/1oNxfvx5jOfcmk1Nx0qavjLN8KtWcLRn6?usp=sharing), even with OpenVINO conversion to .blob)

raymondlo84 commented:
WOW exciting. Just read the code and looks pretty clean and easy to understand.

raymondlo84 commented:
Ok, tested it and yes the result seems off. Maybe let's double-check how the blob is generated (does these look good to you).


`
# Setup model optimizer command ...
ir_name = "flower"
ir_data_type = "FP32"
ir_out_dir = f"{model_fname}/IR_models/{ir_data_type}"
ir_input_shape = "[1,180,180,3]"

mo_cmd = f"/opt/intel/openvino_2021/deployment_tools/model_optimizer/mo_tf.py \
      --saved_model_dir {model_fname} \
      --input_shape {ir_input_shape} \
      --data_type {ir_data_type} \
      --output_dir {ir_out_dir}  \
      --model_name {ir_name}"
print ("Running model optimizer to convert model to OpenVINO IR format ....")
print("\n--".join(mo_cmd.split("--")))

output = subprocess.check_output(source_cmd+" && "+mo_cmd, shell=True)
print (output.decode('utf-8'))   

`
`
!/opt/intel/openvino_2021/deployment_tools/inference_engine/lib/intel64/myriad_compile -m /content/flower/IR_models/FP32/flower.xml -ip U8 -VPU_NUMBER_OF_SHAVES 4 -VPU_NUMBER_OF_CMX_SLICES 4
`

Did we make a mistake there? I also assume the vector returned by the network wont' get flipped or anything?

VanDavv commented:
@raymondlo84 I've added this as [an issue](https://github.com/luxonis/depthai-experiments/issues/55) to be solved, I'm also thinking that it's the conversion issue.

I think that we can merge this, as I added a note about detection issue, and folks may be interested in using this source code with other networks or even take a stab to resolve this issue quicker than us
